jabsrv/src/main/java/org/jabref/http/server/cayw/format/BibLatexFormatter.java

```diff
@@ -17,6 +18,11 @@ public String getFormatName() {
         return "biblatex";
     }
 
+    @Override
+    public MediaType getMediaType() {
+        return MediaType.TEXT_PLAIN_TYPE;
```

Comment:
`application/x-biblatex`?

Source: https://cnb.cool/8888/github/Goldziher/kreuzberg/-/git/raw/wmen_main/kreuzberg/_mime_types.py

Maybe simpler `application/x-bibtex`  is also OK :)